src/plugins/inputs/msg.ts

```diff
@@ -99,7 +98,7 @@ exports.input = function (network, chan, cmd, args) {
 			targetGroup = parsedTarget.target_group;
 		}
 
-		const channel = network.getChannel(targetName);
+		const channel = network.getChannel(String(targetName));
```

Comment:
Just specify string return type for getTarget.

Reply:
getTarget can returned undefined due to using `.shift()`

<img width="724" alt="image" src="https://user-images.githubusercontent.com/8675906/172556878-bf708f0d-2bef-4471-a6a8-d245d32de247.png">

Reply:
Well we do have `if (args.length === 0 || !targetName) {` check, does typescript not understand the flow that it will always be a string here?